Tests/Magick.NET.Tests/Shared/Profiles/Exif/ExifValueTests/TheTrySetValueMethod.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
```

Comment:
I think we can add tests when we have decided how we should add this.

And thanks for following the correct unit test structure 👍

Reply:
👍 

I'm having real problems getting the code to build, though. At the moment the Magick.NET.Native project isn't building because of a link failure, and the Magick.NET project doesn't recognise simple types (e.g. System.bool). The Magick.NET project builds, presumably because of the automatic binding redirects? But if I'm in code view, everything is underlined red.

I've followed the notes in building.md, but not getting anywhere at the moment. Any thoughts?